.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor/pkg
-runc
+/runc
+Godeps/_workspace/src/github.com/opencontainers/runc
```

Comment:
As @kofalt said, I think it's better to specify files we need to ignore, in case we have other files named runc in the future.